contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+a release due to incompleteness. In a blog post describing the [road to Go 2][],
+Russ Cox explains
+
+> that it is difficult but essential to describe the significance of a problem
+> in a way that someone working in a different environment can understand
+
+unfortunately I believe that our current mechanism for proposing changes by
+opening a GitHub issue against `kubernetes/features` falls short of providing
+an effective vehicle for agreeing on the motivation and the design of a proposed
+change.
+
+The use of GitHub issues when proposing changes does not provide SIGs good
+facilities for signaling approval or rejection of a proposed change to Kubernetes
+since anyone can open a GitHub issue at any time. Additionally managing a proposed
+change across multiple releases is somewhat cumbersome as labels and milestones
+need to be updated for every release that a change spans which leads to an ever
+increasing number of issues open against `kubernetes/features` which itself has
+become a management problem.
+
+We have chosen to use Git as our VCS with GitHub currently used to host our
+repositories. While GitHub provides a rich feature set as an open source project,
+managing issues over time and searching for text within an issue can be
+challenging. The flat hierarchy of issues can also make navigation and
+categorization tricky. While not all community members might not be comfortable
+using Git directly, I believe it is imperative that as a community we work to
+educate people on a standard set of tools so they can take their experience to
+other projects they may decide to work on in the future.
+
+It is my hope that by increasing the requirements for proposing a change to
+Kubernetes we can prevent important work like generating release notes from
+falling out of the release cycle and in general make working with and speaking
+about Kubernetes an easier and more pleasant experience.
+
+[road to Go 2]: https://blog.golang.org/toward-go2
+
+## Detailed Design
+
+### RFC Template
+
+### Git and GitHub Implementation
+
+Practically an RFC would be implemented as a pull request to a central repository
+with the following example structure
+
+```
+├── 0000-rfc-template.md
+├── CODEOWNERS
+├── index.md
+├── sig-architecture
+├── sig-network
+│   └── kube-dns
+├── sig-node
+│   └── kubelet
+├── sig-release
+├── sig-storage
+├── unsorted-to-be-used-by-newcomers-only
+└── wg-resource-management
+```
+
+where each SIG or working group is given a top level directory with components
+maintained by the SIG listed in sub directories. For newcomers to the community
+an `unsorted-to-be-used-by-newcomers-only` directory may be used before an RFC
+can be properly routed to a SIG although hopefully if discussion for a potential
+RFC begins on the mailing lists proper routing information will be provided to
+the RFC author. Additionally a top level index of RFCs may be helpful for people
```

Comment:
More than that, it is worthwhile, IMO, to have an interactive index that can help filter proposals based on status and area.  We'll want to cross index based on related proposals, involved SIGs, superseded proposals, etc.